src/Sylius/Component/Attribute/Configure/OptionListAttributeFormOptionConfigurer.php

```diff
@@ -0,0 +1,51 @@
+<?php
+/**
+ * @author    Fernando Caraballo Ortiz <fernando.ortiz@reiss.com>
+ * @date      01/12/2016
+ * @copyright Copyright (c) Reiss Clothing Ltd.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Component\Attribute\Configure;
+
+use Sylius\Bundle\AttributeBundle\Form\Type\AttributeType\Configuration\OptionListAttributeConfigurationType;
+use Sylius\Component\Attribute\Model\AttributeInterface;
+use Sylius\Component\Attribute\Model\AttributeValueInterface;
+
+/**
+ * @author Fernando Caraballo Ortiz <fernando.ortiz@reiss.com>
+ */
+class OptionListAttributeFormOptionConfigurer
+{
+    /**
+     * @param AttributeInterface           $attribute
+     * @param AttributeValueInterface|null $attributeValue
+     *
+     * @return array
+     */
+    public function addOptionsToOptionListType(AttributeInterface $attribute, AttributeValueInterface $attributeValue = null)
```

Comment:
hmm I think either this method needs to be renamed to `getOptionsToOptionListType()` or you need to pass in and return `$options`, because currently this method doesn't add anything to anything.